client/src/components/Background.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+import StartBackground from '../assets/background.svg';
+
+const Background = styled.div`
```

Comment:
background for what and what kind of background?

Reply:
I will fix this with two div elements in the next PR ;)